sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -301,15 +324,18 @@
      * <p><strong>Code sample</strong></p>
      * {@codesnippet com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeReceiptsFromUrl#string}
      *
-     * @param sourceUrl The source URL to the input document. Size of the file must be less than 50 MB.
+     * @param receiptUrl The source URL to the input document. Size of the file must be less than 50 MB.
```

Comment:
nit: "The source URL to the input receipt." 

Also unsure if size of the file is as necessary for the FromUrl functions